ball:
  ball_topic: '/balls_in_image'
  # ballradius in meter
  ball_radius: 0.075

lines:
  lines_topic: '/line_in_image'
  # if the line_in_image message is used to only to sent points and a point cloud should be published
  lines_relative: true
  pointcloud: true

goals:
  goals_topic: '/goal_in_image'

goal_parts:
  goal_parts_topic: '/goal_parts_in_image'
  bar_height: 2.0

obstacles:
  obstacles_topic: '/obstacles_in_image'

field_boundary:
  field_boundary_topic: '/field_boundary_in_image'

camera_info:
  camera_info_topic: '/camera_info'

